cpp/core/jni/JniWrapper.cc

```diff
@@ -354,11 +356,10 @@ JNIEXPORT void JNICALL Java_io_glutenproject_vectorized_PlanEvaluatorJniWrapper_
   JNI_METHOD_START
 
   auto len = env->GetArrayLength(path);
-  jbyte* bytes = env->GetByteArrayElements(path, 0);
-  std::string pathStr(reinterpret_cast<char*>(bytes), len);
+  auto safeArray = gluten::getByteArrayElementsSafe(env, path);
+  std::string pathStr(reinterpret_cast<char*>(safeArray.elems()), len);
   auto ctx = gluten::getRuntime(env, wrapper);
   ctx->injectWriteFilesTempPath(pathStr);
-  env->ReleaseByteArrayElements(path, bytes, JNI_ABORT);
```

Comment:
does this mean the `env->ReleaseByteArrayElements` not work as expected?

Reply:
It should work but some of our calls to `Get` doesn't have a corresponding call to `Release`. So this code was a good case not leading to memory leak.

Reply:
As a comparison the code in `nativeCreateKernelWithIterator` may leak considerable amount of memory since it's called frequently and missed several `Release` calls.